web/src/settings_invites.ts

```diff
@@ -254,11 +258,14 @@ export function on_load_success(
         };
         const html_body = render_settings_revoke_invite_modal(ctx);
 
-        confirm_dialog.launch({
+        dialog_widget.launch({
```

Comment:
I guess you changed this as `close_on_submit` is passed as `true` and we do not want the modal to close immediately. I think you can just pass `close_on_submit`, and other arguments like `id` as well, when using `confirm_dialog` as well.

Reply:
Yes, you are right about the reason for using `dialog_widget` instead of `confirm_dialog`.
I followed the approach mentioned earlier, but it caused the modal to remain open. I’ve included a screen recording demonstrating the issue with `confirm_dialog`, where `close_on_submit` is passed as `true` and no errors are encountered. Under normal circumstances, the modal should close, but it does not.
<details>
<summary>Screen Recording</summary>

![revoke test](https://github.com/user-attachments/assets/a4243cfb-c6f0-47d9-8f53-8467983d0774)

</details>

Please note that in the above recording, the invite was revoked, but the modal only closed when I clicked the Cancel button. The same case applies to resending an invitation.

Reply:
Ok, I get this now. It is not the issue with using `confirm_dialog` or `dialog_widget`. When passing `close_on_submit: false`, you need to manually handle closing of modal, which you are doing.

Reply:
Can we not use `confirm_dialog` as before?